src/fu-engine.c

```diff
@@ -8857,15 +8856,6 @@ fu_engine_constructed(GObject *obj)
 #endif
 
 	fu_context_add_compile_version(self->ctx, "org.freedesktop.fwupd", VERSION);
-#ifdef HAVE_GUSB
-	{
-		g_autofree gchar *version = g_strdup_printf("%i.%i.%i",
-							    G_USB_MAJOR_VERSION,
-							    G_USB_MINOR_VERSION,
-							    G_USB_MICRO_VERSION);
-		fu_context_add_compile_version(self->ctx, "org.freedesktop.gusb", version);
```

Comment:
anything similar for libusb we should be keeping in place?  Or API is stable enough really that we don't care?

Reply:
Ahh, we can get the compile version from the pkgconf file -- fixed.